cmd/testdata/help.ct

```diff
@@ -1,41 +1,53 @@
 # shows help output when no arguments are specified
 $ gql-lint
-Usage: gql-lint <command> [<args>]
+gql-lint is a tool to lint GraphQL queries and mutations
```

Comment:
I would remove these tests, or maybe just leave this first one to make sure the message doesn't get broken by accident

Reply:
I guess with Cobra there's less need as help is more automatic that with `flags`. The tests were added to ensure we output help at the correct times as there's many variants that can display help output and hence easy to break one of them: `cmd --help` vs `cmd help` vs `cmd sub --help` vs `cmd sub help` etc.

With `go-cmdtest` it's easy enough to regenreate the output so I don't see much downside in keeping it...